pkg/operator/ceph/cluster/watcher.go

```diff
@@ -343,7 +348,7 @@ func listRWOCephFSPV(listPVs *corev1.PersistentVolumeList, cluster *cephv1.CephC
 				continue
 			}
 
-			if pv.Spec.CSI.VolumeAttributes["staticVolume"] == "true" || pv.Spec.CSI.VolumeAttributes["pool"] == "" {
+			if pv.Spec.CSI.VolumeAttributes["staticVolume"] == "true" || pv.Spec.CSI.VolumeAttributes["fsName"] == "" {
```

Comment:
we dont need to check `fsName` for a static volume if `staticVolume` check is enough, i don't see when fsName can be set to empty.

Reply:
So we can simplify to this:
```suggestion
			if pv.Spec.CSI.VolumeAttributes["staticVolume"] == "true" {
```